chore(react-scheduler): add Redux Integration Demo (#2304)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/demos/featured/redux-integration.md

> # React Scheduler Redux Integration

Reviewer: ```suggestion
# React Scheduler - Redux Integration
```

Reviewer 2: We often don't use `-` inside the `Demos` section https://devexpress.github.io/devextreme-reactive/react/grid/demos/featured/remote-data/

Reviewer: Docs use but demos don't. That's interesting.

> In this demo, the Scheduler works as a stateless component while the [Redux store](http://redux.js.org/docs/basics/Store.html) manages its state. All end-user actions that modify the component state produce corresponding [Redux actions](http://redux.js.org/docs/basics/Actions.html) the Scheduler [reducer](http://redux.js.org/docs/basics/Reducers.html) dispatches. The [Redux DevTool Extension](https://github.com/zalmoxisus/redux-devtools-extension#redux-devtools-extension) is activated in this demo, which allows you to perform time-traveling and observe Scheduler state changes.

Reviewer: ```suggestion
In this demo, the Scheduler works as a stateless component, with the [Redux store](http://redux.js.org/docs/basics/Store.html) managing its state. All user actions that modify the state produce corresponding [Redux actions](http://redux.js.org/docs/basics/Actions.html) dispatched by the Scheduler's [reducer](http://redux.js.org/docs/basics/Reducers.html). Thanks to the [Redux DevTool Extension](https://github.com/zalmoxisus/redux-devtools-extension#redux-devtools-extension) activated in this demo, you can perform time-traveling and observe how the Scheduler's state was changing.
```

> In this demo, the Scheduler works as a stateless component, with the [Redux store](http://redux.js.org/docs/basics/Store.html) managing its state. All user actions that modify the state produce corresponding [Redux actions](http://redux.js.org/docs/basics/Actions.html) dispatched by the Scheduler's [reducer](http://redux.js.org/docs/basics/Reducers.html). Thanks to the [Redux DevTool Extension](https://github.com/zalmoxisus/redux-devtools-extension#redux-devtools-extension) activated in this demo, you can perform time-traveling and observe how the Scheduler's state was changing.

Reviewer 3: ```suggestion
In this demo, the Scheduler is used as a stateless component and the [Redux store](http://redux.js.org/docs/basics/Store.html) manages its state. User actions that modify the state have corresponding [Redux actions](http://redux.js.org/docs/basics/Actions.html) dispatched by the Scheduler's [reducer](http://redux.js.org/docs/basics/Reducers.html). The [Redux DevTool Extension](https://github.com/zalmoxisus/redux-devtools-extension#redux-devtools-extension) in this demo shows changes to the Scheduler's state.
```

Reviewer: ```suggestion
In this demo, the Scheduler is used as a stateless component, and the [Redux store](http://redux.js.org/docs/basics/Store.html) manages its state. User actions that modify the state have corresponding [Redux actions](http://redux.js.org/docs/basics/Actions.html) dispatched by the Scheduler's [reducer](http://redux.js.org/docs/basics/Reducers.html). If the [Redux DevTool Extension](https://github.com/zalmoxisus/redux-devtools-extension#redux-devtools-extension) is installed in your browser, you can view how the Scheduler's state changes.
```
